infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -1361,10 +1394,10 @@ public void reset() {
 
         serverContext.setStartingAddress(0L);
         serverContext.setTailSegment(0L);
-        globalTail.set(Address.NON_ADDRESS);
-        initializeStartingAddress();
-        initializeMaxGlobalAddress();
-
+        startingAddress = 0L;
+        lastSegment = 0L;
+        logMetadata = new LogMetadata();
+        writeChannels.clear();
```

Comment:
Do we need to call `initializeLogMetadata`?

Reply:
There's nothing to initialize after the reset.